backend/runprocessors.py

```diff
@@ -44,6 +44,12 @@
     help="program, specialisation, course, condition, data-fix, enrolment",
 )
 
+parser.add_argument(
+    "--year",
```

Comment:
Not needed, there is processors_archive.py. either use that or delete that file since it will now be redundant

Reply:
Wait we have that? Why didn't u tell me that earlier 😭